docs/docs/genai/data-model/app-versions.mdx

```diff
@@ -0,0 +1,564 @@
+# Version Tracking Data Model
```

Comment:
Should we also include Model Version (model registry) in this data-model section? 
It could be the same as the classicML one, but if we don't have it in GenAI doc then we can't link it (e.g. when we say serving on Databricks we'll need to mention that the model should be registered)

Reply:
Added :)